tests/plugins/test_koji_promote.py

```diff
@@ -1230,31 +1201,27 @@ def test_koji_promote_success(self, tmpdir, apis, docker_registry,
             assert len([b for b in buildroots
                         if b['id'] == buildroot['id']]) == 1
 
-        if docker_registry:
-            # v2 support
-            if 'v1' in registry_digests and 'v2' in registry_digests:
-                if not pulp_registries:
-                    expect_digest = (2,)
+        # v2 support
+        if 'v1' in registry_digests and 'v2' in registry_digests:
+            if not pulp_registries:
+                expect_digest = (2,)
+            else:
+                if pulp_supports_schema2:
+                    expect_digest = (1, 2)
                 else:
-                    if pulp_supports_schema2:
-                        expect_digest = (1, 2)
+                    if sys.version_info.major > 2:
```

Comment:
I think this could be an `elif` chain instead of nested ifs:
```python
if not pulp_registries:
    expect_digest = (2,)
elif pulp_supports_schema2:
    expect_digest = (1, 2)
elif sys.version_info.major > 2:
    expect_digest = (2,)
else:
    expect_digest = (1,)
```

Reply:
Thanks, fixed.